app/code/Magento/Cms/Api/BlockRepositoryInterface.php

```diff
@@ -32,6 +32,16 @@ public function save(Data\BlockInterface $block);
     public function getById($blockId);
 
     /**
+     * Load Block data by given Block identifier
+     *
+     * @param string $blockIdentifier
+     * @param int $storeId
+     * @return \Magento\Cms\Api\Data\BlockInterface
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
+     */
+    public function getByIdentifier($blockIdentifier, $storeId);
```

Comment:
What do you suggest? Should I simply remove this part from the interface and let the implementation in the model ( `app/code/Magento/Cms/Model/ResourceModel/Block.php` ) ?